### ISSUE TITLE
[Accessibility][P2] Token cannot be deleted by keyboard

### ISSUE THREAD

**Issue by Slevunio** [SAP/fundamental-ngx#13615]
### Describe your proposal

We're using [TokenComponent](https://sap.github.io/fundamental-ngx/#/core/token) as part of our feature. 
Accessibility issue was reported regarding deleting the token with keyboard usage only. **It is impossible to navigate to delete button within token with keyboard only.**

### Which versions of Angular and Fundamental Library for Angular are affected?

Angular: 19.2.14, fundamentals-ngx/core: 0.56.5

As 0.56 is LTS version, could you please provide a fix in this version too?

### Steps to Reproduce (for bugs)

1. Go to official component's page: [](https://sap.github.io/fundamental-ngx/#/core/token)
2. Navigate to any token in demo page with tab button
3. Try to focus on delete button within the token

Actual:
It is impossible to focus on delete button within the token therefore user cannot delete it without mouse usage.

Expected:
It should be possible to navigate within token with keyboard only.

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by droshev** [comment]
@Slevunio did you use tokenizer like [here](https://fundamental-ngx.netlify.app/#/core/token#tokenizer) ?

**Comment by Slevunio** [comment]
@droshev 
Nope. Tokens are used as popover controllers. Smth like:
`<fd-popover>
    <fd-popover-control>
        <fd-token (click)="doSmth()"> 
            ....
        </fd-token>
    </fd-popover-control>
<fd-popover>`

**Comment by droshev** [comment]
you would need a wrapper component to handle those functionality(keyboard) like the tokenizer.

**Comment by Slevunio** [comment]
This is a part of shared package component that is already used by many apps. Providing such a wrapper would change UI -> I'm not sure if this an option.

**Comment by droshev** [comment]
@Slevunio You would need a wrapper where to handle the keyboard, as we do in tokenizer
